### PR TITLE
Add lesson 3

### PR DESCRIPTION
**Pull request by EugeniyBurmistrov**

### CODE REVIEW

File: docs/basics/tutorial/lesson-3.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+## **Lesson 3** - Read side - Create a View Model to Query List Items
+
+[\[Get the Code for This Lesson\]](https://github.com/reimagined/resolve/tree/master/examples/shopping-list-tutorial/lesson-3)
+
+Currently, your shopping list application has a write side that allows you to create shopping lists and items in these lists. To provide the capability to obtain this data from the application, you need to implement the application's **[read side](resolve-app-structure.md#write-and-read-sides)**.
```
Comment (dirk-pieterse) [inline docs/basics/tutorial/lesson-3.md]:
```suggestion
Currently, your shopping list application has a write side that allows you to create shopping lists and add items to these lists. To provide the capability to obtain this data from the application, you need to implement the application's **[read side](resolve-app-structure.md#write-and-read-sides)**.
```

Comment (dirk-pieterse) [reply docs/basics/tutorial/lesson-3.md]: "To provide the capability to" is a very long way to say 'to'.

File: docs/basics/tutorial/lesson-3.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        checked: false
+      }
+    ]
+  })
+};
+```
+
+<!-- prettier-ignore-end -->
+
+This code defines a View Model **[projection](read-side.md#updating-a-read-model-via-projection-functions)**. A View Model projection runs when the View Model receives a data query. It runs for all events with the specified aggregate IDs and builds state based on data from these event. The resulting state is then sent back as the query response.
```
Comment (dirk-pieterse) [inline docs/basics/tutorial/lesson-3.md]: "and builds state" a state? the state?

Comment (EugeniyBurmistrov) [reply docs/basics/tutorial/lesson-3.md]: changed to 'a state'

File: docs/basics/tutorial/lesson-3.md (hunk 1 above)
Comment (EugeniyBurmistrov) [reply docs/basics/tutorial/lesson-3.md]: Changed to 'to'

**Pull request merged**